test/fixtures/promises/1-callback-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+module.exports.up = function (next) {
+  setTimeout(function () {
+    return next()
+  }, 200)
+}
+
+module.exports.down = function (next) {
+  setTimeout(function () {
+    return next()
+  }, 200)
```

Comment:
To reduce test time, just do a timeout of `1` ms.  It achieves the same effect without the wait.